Projects docs (#237)
hashicorp/terraform-docs-common: website/docs/cloud-docs/overview.mdx

Remote execution can be disabled on specific workspaces with the ["Execution Mode" setting][execution_mode]. The workspace will still host remote state, and Terraform CLI can use that state for local runs via the [Terraform Cloud CLI integration](/cli/cloud).

## Organize Infrastructure with Projects and Workspaces

> Terraform's local workflow manages a collection of infrastructure with a persistent working directory, which contains configuration, state data, and variables. You can use separate directories to organize infrastructure resources into meaningful groups, and Terraform will use content from whichever directory it is invoked from.

Reviewer: ```suggestion
Terraform's local workflow manages a collection of infrastructure with a persistent working directory, which contains configuration, state data, and variables. You can use separate directories to organize infrastructure resources into meaningful groups, and Terraform you invoke Terraform commands from.
```

> Terraform Cloud organizes infrastructure with projects and workspaces instead of directories. Each workspace contains everything necessary to manage a given collection of infrastructure, and Terraform uses that content when it runs in the context of that workspace.

Reviewer: ```suggestion
Terraform Cloud organizes infrastructure into projects and workspaces instead of directories. Each workspace contains everything necessary to manage a given collection of infrastructure, and Terraform uses that content when it runs in the context of that workspace.
```

> You can further organize workspaces into projects. Organizations with the Team & Governance tier can scope access to specific projects, limiting access to only the resources required for a team member's job function. 

Reviewer: ```suggestion
You can use projects to organize your workspaces into groups. Organizations with the [Team & Governance](https://www.hashicorp.com/products/terraform/pricing) tier can assign teams permissions for specific projects. 
This lets you grant access to collections of workspaces instead of using workspace-specific or organization-wide permissions, making it easier to limit access to only the resources required for a team member's job function. 
```
